products/sle/main.pm

```diff
@@ -907,6 +907,8 @@ elsif (get_var('LIBSOLV_INSTALLCHECK')) {
 }
 elsif (get_var("EXTRATEST")) {
     boot_hdd_image;
+    loadtest "console/docker";
+    loadtest "console/zypper_docker";
```

Comment:
We already have a function load_extra_tests_docker in https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/lib/main_common.pm#L1531 . Do you think your module could fit there as well?

Reply:
yes sure ;)